app/build.gradle

```
@@ -55,7 +60,17 @@ dependencies {
     // Coil
     implementation "io.coil-kt:coil:1.1.1"
 
+    // Navigation
     def nav_version = "2.5.1"
     implementation "androidx.navigation:navigation-fragment-ktx:$nav_version"
     implementation "androidx.navigation:navigation-ui-ktx:$nav_version"
+
+    // Paging 3.0
+    def paging_version = "3.0.0"
+    implementation "androidx.paging:paging-runtime:$paging_version"
+    testImplementation "androidx.paging:paging-common:$paging_version"
+    implementation "androidx.paging:paging-rxjava2:$paging_version"
+    implementation "androidx.paging:paging-rxjava3:$paging_version"
+    implementation "androidx.paging:paging-guava:$paging_version"
+    implementation "androidx.paging:paging-compose:1.0.0-alpha08"
```

```suggestion
    def paging_version = "3.0.0"
    implementation "androidx.paging:paging-runtime:$paging_version"
    testImplementation "androidx.paging:paging-common:$paging_version"
```
필요한 dependency만 추가하는게 좋아보입니다ㅎㅎ